github/apps_marketplace.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// MarketplaceService handles communication with the marketplace related
+// methods of the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/marketplace/
+type MarketplaceService service
+
+// MarketplacePlan represents a GitHub Apps Marketplace Listing Plan.
+type MarketplacePlan struct {
+	URL                 *string   `json:"url"`
+	AccountsURL         *string   `json:"accounts_url"`
+	ID                  *int      `json:"id"`
+	Name                *string   `json:"name"`
+	Description         *string   `json:"description"`
+	MonthlyPriceInCents *int      `json:"monthly_price_in_cents"`
+	YearlyPriceInCents  *int      `json:"yearly_price_in_cents"`
+	PriceModel          *string   `json:"price_model"`
+	UnitName            *string   `json:"unit_name"`
+	Bullets             *[]string `json:"bullets"`
+}
+
+// MarketplacePurchase represents a GitHub Apps Marketplace Purchase.
+type MarketplacePurchase struct {
+	BillingCycle    *string          `json:"billing_cycle"`
+	NextBillingDate *string          `json:"next_billing_date"`
+	UnitCount       *int             `json:"unit_count"`
+	Plan            *MarketplacePlan `json:"plan"`
+}
+
+// MarketplacePlanAccount represents a GitHub Account (user or organization) on a specific plan
```

Comment:
In general, I think we should strive to have all sentences in Go documentation to have no spelling or grammar mistakes, and to include periods at ends of sentences. So, I'll point out that you're missing a period at the end of this sentence.